apps/templates/includes/sidebar.html

```diff
@@ -81,8 +81,15 @@
               </p>
             </a>
           </li>
-          {% if current_user.category == "admin" or current_user.category == "teacher" %}
           <li class="nav-header">MANAGEMENT</li>
+          <li class="nav-item">
+            <a href="{{ url_for('home_blueprint.view_group', group_id=None) }}" 
+               class="nav-link {% if request.url_rule.endpoint == 'home_blueprint.view_group' %} active {% endif %}">
+              <i class="nav-icon fas fa-users"></i>
+              <p>Groups</p>
+            </a>
+          </li>
+          {% if current_user.category == "admin" or current_user.category == "teacher" %}
```

Comment:
I wouldn't call the Groups page as a management action. Can you pleas move it to the same group of the main functions (like Running Labs, Labs, etc).